_posts/cci/2021-08-20-cci-download-stats.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "What is a user? Removing anomalous behaviour from Anonymous access logs."
+author: Mahir Rahman
+date:   2021-08-20 17:00:00
+tags: CCI, download stats
+---
+
+The Climate Change Initiative (CCI) [project's goal](https://climate.esa.int/en/esa-climate/esa-cci/Objective/) is to provide open, registration free, 
+access to essential climate variables (ECVs). Dataset usage is an important metric in understanding uptake and usage if the different datasets however,
+without requiring users to register, it is difficult to determine distinct users. Recent changes in access patterns have led to spurious user counts when
+thinking 1 IP = 1 USER. This article looks at methods to determine "normal" thresholds to reduce the impact of the different access patterns on our usage statistics.
+
+# Index
+
+- [Index](#index)
+- [Background](#background)
+- [Filtering](#filtering)
+- [Conclusion](#conclusion)
+
+# Background
+
+Download and access stats for the CCI Open Data Portal (ODP) is important to track which data is being used and report back to 
+both the scientists producing the data and the funding bodies who comission the data and storage. It allows us to see 
+trends and inform decisions about promotional activities, but counting users from anonymous logs is difficult.
+
+There are multiple, legitimate uses of the data but not all count towards usage statistics. One obvious usage which should
+be removed for reporting is usage stats generated through development activity of downstream projects. The CCI ODP is used
+by multiple projects as part of the CCI Knowledge Exchange project. This, in itself, presents a challenge as users often have
+dynamic IPs and IP ranges can filter out non-development usage.
+
+The other challenge is different access patterns. Traditionally, users use a single machine to download data. This may present itself as a number
+of IPs (due to dynamic IP assignment) but change infrequently. One access pattern we have seen in the last year is a large number of IP adresses 
+accessing the same dataset within a single day or two, by one download method. Looking at the usage and the metadata in the logs, it is likely
+(not possible to know without registered user data) that this is a single user using a novel access pattern, possibly torrent. This access
+pattern artificially bloats the number of users.
+
+**insert graph showing anomalous access pattern**
```

Comment:
Add a graph of 2021 just showing the spikes. Make sure to give the graph a title and axis labels.